agency/README.md

```diff
@@ -101,68 +102,29 @@ Body Params:
+The vehicle `/telemetry` endpoint allows a Provider to update vehicle telemetry data in batch for one or many of the vehicles in the fleet. Telemetry data will be reported to the API every 5 seconds while vehicles are in motion.
 
-A trip represents a route taken by a provider's customer.   Trip data will be reported to the API every 5 seconds while the vehicle is in motion.
+Endpoint: `/vehicles/telemetry`
+Method: `POST`
 
-Endpoint: `/update_trip_telemetry`\
-Method: `POST`\
-API Key: `Required`
+Body Params:
 
-Body:
+| Field         | Type                           | Required/Optional | Field Description                                                                      |
+| ------------- | ------------------------------ | ----------------- | -------------------------------------------------------------------------------------- |
+| `data`        | [Telemetry](#telemetry-data)[] | Required          | Array of telemetry for one or more vehicles.                                           |
 
-| Field | Type     | Required/Optional | Other |
-| ----- | -------- | ----------------- | ----- |
-| `trip_id` | UUID | Required | Issued by InitMovementPlan() API  |
-| `timestamp` | Unix Timestamp | Required | Time of day (UTC) data was sampled|
-| `route` | Route | Required | See detail below. |
-| `accuracy` | Integer | Required | The approximate level of accuracy, in meters, represented by start_point and end_point. |
+201 Success Response:
 
-Response:
+_No content returned on success._
 
-| Field | Type | Other |
-| ---- | --- | --- |
-| `message` | Enum | See [Message](#message) Enum |
-
-### Route
-
-To represent a route, MDS provider APIs should create a GeoJSON Feature Collection where ever observed point in the route, plus a time stamp, should be included. The representation needed is below.
-
-The route must include at least 2 points, a start point and end point. Additionally, it must include all possible GPS samples collected by a provider. All points must be in WGS 84 (EPSG:4326) standard GPS projection
-
-```js
-"route": {
-        "type": "FeatureCollection",
-        "features": [
-            {
-                "type": "Feature",
-                "properties": {
-                    "timestamp": 1529968782.421409
-                },
-                "geometry": {
-                    "type": "Point",
-                    "coordinates": [
-                        -118.46710503101347,
-                        33.9909333514159
-                    ]
-                }
-            },
-            {
-                "type": "Feature",
-                "properties": {
-                    "timestamp": 1531007628.3774529
-                },
-                "geometry": {
-                    "type": "Point",
-                    "coordinates": [
-                        -118.464851975441,
-                        33.990366257735
-                    ]
-                }
-            }
-        ] }
-```
+400 Failure Response:
```

Comment:
This should probably not be part of a specific section but rather be defined globally for the agency API.

Reply:
The responses are specific to each endpoint.  The codes are global.